### DISCUSSION TITLE
Error ketika install requirements.txt

### DISCUSSION THREAD

**Discussion by AnthonySSSSS** [teamalgoritma/community#356] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/112844100/192563464-24478b80-6020-4f0a-8f44-85c618f8ff46.png)

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @AnthonySSSSS , Bapak boleh coba meng-copy seluruh code berwarna merah di line keduanya saja kemudian run kembali. Bila tetap tidak bisa, copy dari bagian `python.exe -m pip install -r requirements.txt` saja. Semoga membantu!
